src/testRunner/runner.ts

```diff
@@ -151,11 +151,21 @@ function handleTestConfig() {
         }
 
         if (testConfig.stackTraceLimit === "full") {
-            (Error as any).stackTraceLimit = Infinity;

```

Comment:
We probably don't need to change this for the test runner since that isn't shipping code.